packages/qwik/src/qwikloader.ts

```diff
@@ -139,8 +132,9 @@ export const qwikLoader = (doc: Document, hasInitialized?: number, prefetchWorke
     }
   };
 
-  const addDocEventListener = (eventName: string) =>
-    doc.addEventListener(eventName, processEvent, { capture: true });
+  const addDocEventListener = (eventName: string) => {
+    window.addEventListener(eventName, processWindowEvent, { capture: true });
```

Comment:
This line breaks listening on `mousenter`. 

My theory: `mouseenter` is a non-bubbling event. This is the reason why we have to do `capture: true`. However, because it is non-bubbling it never gets to the `window` it only stays on the document. So I think this breaks the code.